docs/book/SUMMARY.md

```diff
@@ -1,4 +1,8 @@
 # Summary
 
 * [Introduction](README.md)
+* [Prerequisites](prerequisites.md)
+* [Getting Started](getting_started.md)
+* [Service LoadBalancers](service_loadbalancers.md)
+* [Known Limitations](known_limitations.md)
```

Comment:
What else do we need to add here?

Reply:
EBS is going off to CSI now?
IAM perms is usually the key thing.

Reply:
Actually, tags might be helpful to document.

Reply:
> IAM perms is usually the key thing.

Agreed, I think this would go into the Prerequisites section.

Reply:
> Actually, any default tags the CPI set might be helpful to document.

Wondering what section this would belong to.

Reply:
sgtm